src/Examples/DAE/petsc_pid_example.ipynb

```diff
@@ -369,10 +347,10 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "2022-04-20 16:38:00 [INFO] idaes.init.fs.valve_1: Initialization Complete: optimal - Optimal Solution Found\n",
-      "2022-04-20 16:38:00 [INFO] idaes.init.fs.tank.control_volume: Initialization Complete\n",
-      "2022-04-20 16:38:00 [INFO] idaes.init.fs.tank: Initialization Complete: optimal - Optimal Solution Found\n",
-      "2022-04-20 16:38:00 [INFO] idaes.init.fs.valve_2: Initialization Complete: optimal - Optimal Solution Found\n"
+      "2022-04-21 14:41:23 [INFO] idaes.init.fs.valve_1: Initialization Complete: optimal - Optimal Solution Found\n",
+      "2022-04-21 14:41:23 [INFO] idaes.init.fs.tank.control_volume: Initialization Complete\n",
+      "2022-04-21 14:41:23 [INFO] idaes.init.fs.tank: Initialization Complete: optimal - Optimal Solution Found\n",
+      "2022-04-21 14:41:23 [INFO] idaes.init.fs.valve_2: Initialization Complete: optimal - Optimal Solution Found\n"
```

Comment:
It would be nice if we could set up Git so that it didn't show changes to captured output from Jupyter notebooks by default. Do @ksbeattie or @lbianchi-lbl know how?